### PR TITLE
[OPS-698] Include DB schema in distribution, fix example config

### PR DESCRIPTION
**Pull request by kirelagin**
### Description

This PR contains changes required for deploying Educator alongside a postgres database:

* Fix example config
* Add .sql files to data-files in educator
* Provide a derivation with data outputs
* Export all local packages from release.nix

### YT issue

https://issues.serokell.io/issue/OPS-698

### Checklist

Hint: a perfect PR has all the checkmarks set.

Possible related changes (conditional):
- [x] I added tests if required, in case they are:
  - Covering new introduced functionality (feature).
  - Regression tests preventing the bug from silently reappearing again (bugfix).
- [x] I have checked the [sample config](../tree/master/docs/config-full-sample.yaml) and [launch scripts](../tree/master/scripts/launch) and updated them if it was necessary (especially if executables or CLIs were changed).
- [x] I have checked READMEs and changed them accordingly if it's necessary (the main [README.md](../tree/master/README.md) as well as subproject READMEs for all related executables).
- [x] If a public API structure or/and data serialization was changed, I made sure that these changes are reflected in the:
  - [Swagger](../tree/master/specs/disciplina) API specs.
  - Any [related documentation](../tree/master/docs/api-types.md).
- [x] If any public documentation was written, I have spellchecked it.

Stylistic (obligatory):
- [x] My commit history is clean, descriptive and do not contain merge or revert commits or commits like "WIP".
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).
- [x] My code is documented (haddock) and these docs are decent (full enough and spellchecked).

### CODE REVIEW

**Comment by pasqu4le** [comment]
P.s. there was a typo in the PR title (`Inculde`) and I fixed it, but the same one is still in the commit message

File: educator/package.yaml [hunk 1]
```diff
@@ -107,3 +107,6 @@ executables:
       - disciplina-educator
       - loot-log
       - optparse-applicative
+
+data-files:
+  - database/*.sql
```
Comment (flyingleafe) [inline educator/package.yaml]: @kirelagin Is it necessary though? `schema.sql` is completely included in the code in compile time via TH, and `fixtures.sql` is used only for manually setting up an Educator database with some data for manual testing. It is necessary only if you want to initialize Postgres via nix, not via leaving it to node itself.

Comment (kirelagin) [reply educator/package.yaml]: Yeah, I initially misread the issue description. It turns out that there is no need to bundle these, so I suggest we close this PR and someone fixes the docs separately.

Comment (pasqu4le) [reply educator/package.yaml]: I already included the config fix in #366

**Comment by flyingleafe** [comment]
Inclusion of SQL schema is unnecessary, config fixes are already introduced.

**Pull request closed without merging**